scripts/run_kickstart_tests.sh

```diff
@@ -153,9 +158,12 @@ function find_tests() {
     local tests=$(find . -maxdepth 1 -name '*sh' -a -perm -o+x)
 
     local newtests=""
+    local skipped_tests=""
     for f in ${tests}; do
         if [[ "$TESTTYPE" != "" && "$(grep TESTTYPE= ${f})" =~ "${TESTTYPE}" ]]; then
             newtests+="${f} "
+        elif [[ "$SKIP_TESTTYPE" != "" && "$(grep TESTTYPE= ${f})" =~ "${SKIP_TESTTYPE}" ]]; then
+            skipped_tests+="${f}"
```

Comment:
If you do this check first, it will be possible to also exclude tests from specified types of tests. For example, you could run all storage tests except for autopart with `-t storage -s autopart`.

Reply:
Good point, could you please make that change before merge.